test/support/test_helpers.ex

```diff
@@ -1,8 +1,10 @@
 defmodule ResearchResource.TestHelpers do
-  alias ResearchResource.Repo
+  alias ResearchResource.{Repo, User}
+  alias Plug.Conn
```

Comment:
@shouston3 is there a risk to use Conn instead of ConnCase in the tests?
```
 This module defines the test case to be used by
  tests that require setting up a connection.
  Such tests rely on `Phoenix.ConnTest` and also
  import other functionality to make it easier
  to build and query models.
  Finally, if the test case interacts with the database,
  it cannot be async. For this reason, every test runs
  inside a transaction which is reset at the beginning
  of the test unless the test case is marked as async.
```
see https://github.com/research-resource/research_resource/blob/2f44f8467b3e5040aeb7ae04bb68412c72126f62/test/support/conn_case.ex#L3-L13